src/neighbor.cpp

```diff
@@ -472,6 +479,18 @@ void Neighbor::init()
       ex_mol_bit[i] = group->bitmask[ex_mol_group[i]];
   }
 
+  if (nex_ivec) {
+//    if (lmp->kokkos)
+//      init_ex_ivec_bit_kokkos();
```

Comment:
Here should probably be an error->all() stating that custom exclusions are not supported with KOKKOS activated.